rpc_jobs/rpc_upgrades.yml

```diff
@@ -248,6 +248,26 @@
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
     CRON: "{CRON_WEEKLY}"
 
+- project:
+    name: "rpc-upgrades-aio-newton-snapshot-incremental"
+    repo_name: "rpc-upgrades"
+    repo_url: "https://github.com/rcbops/rpc-upgrades"
+    image:
+      - xenial_snapshot:
+          REGIONS: "DFW"
+          BOOT_TIMEOUT: 1500
+    scenario:
+      - newton:
+          IMAGE: "rpc-r14-xenial_loose_artifacts-swift"
+      - pike:
+          IMAGE: "rpc-r16-xenial_no_artifacts-swift"
+    action:
+      - "newton_to_queens_incremental"
```

Comment:
The resulting job name doesn't make much sense. Are you sure this is what you want? It seems to me that the scenario/action axes setup you have for the mnaio job above makes a bit more sense?

Reply:
Yeah, the scenario is a bit redundant with the action, but changing the action would break how we parse it currently.  I'm mainly doing this as a test to see if it works and if we benefit from doing snapshots, I'll probably refactor how the scenarios/actions work.

Reply:
Yeah, that makes more sense, I forgot you can move those variables anywhere, updated.